src/server.cpp

```diff
@@ -318,6 +321,67 @@ int Server<IoType, SwitchActivityInfo, SwitchCalcGaps>::server_accept(int ifd) {
                                         inet_ntoa(addr.sin_addr), ntohs(addr.sin_port), active_ifd);
                             }
                             do_accept = true;
+
+                            if (s_user_params.sockperfDumpDataToFile)
+                            {
+                                retVal = system((createSockperfDirectoryCommand.append(s_user_params.sockperfDumpDataDirectory)).c_str());
+                                if(retVal != 0)
+                                {
+                                    log_msg("%s","unable to create directory structure");
+                                }
+                                sockperfDataFile = generateDumpFileName(inet_ntoa(addr.sin_addr));
```

Comment:
When will this file be deleted?

Reply:
This will not be deleted but will always open in overwrite mode.